src/PointEval/PointEvalHandler.jl

```diff
@@ -273,3 +273,41 @@ function get_func_interpolations(dh::DH, fieldname) where DH<:MixedDofHandler
     end
     return func_interpolations
 end
+
+# Iteration of PointEvalHandler
+struct PointIterator{PH<:PointEvalHandler}
+    ph::PH
+    current_idx::ScalarWrapper{Int}
+    nodes::Vector{Int}
+    coords::Vector{Vec{2,Float64}}
+end
+function PointIterator(ph::PointEvalHandler{G}) where {D,C,T,G<:Grid{D,C,T}}
+    @assert isconcretetype(C) # Check mate MixedDofHandler
```

Comment:
```suggestion
    C = getcelltype(grid)
    @assert isconcretetype(C) # Check mate MixedDofHandler
```

is there a benefit of restricting it to `Grid` subtypes in general or can we directly use `<:AbstractGrid` and use the interface?